README.md

```diff
@@ -550,6 +550,7 @@ See also [Are we (I)DE yet?](https://areweideyet.com/) and [Rust Tools](https://
 
   * [Atom](https://atom.io/)
     * [zargony/atom-language-rust](https://github.com/zargony/atom-language-rust)
+    * [rust-lang/atom-ide-rust](https://github.com/rust-lang/atom-ide-rust) — Rust IDE support for Atom, powered by the Rust Language Server (RLS) [![Build Status](https://travis-ci.com/rust-lang/atom-ide-rust.svg?branch=master)](https://travis-ci.com/rust-lang/atom-ide-rust)
```

Comment:
`https://travis-ci.com/rust-lang/atom-ide-rust.svg?branch=master` 301 redirects to `https://api.travis-ci.com/rust-lang/atom-ide-rust.svg?branch=master` so can you please change over the image url to that please?

Reply:
Hi. Sorry it took a while! been busy with work.

Made the change. I hope it works now.